kits/arm/ex_AR_kit.cpp

```diff
@@ -0,0 +1,223 @@
+ * This demo sets up your arm's end effector to mimick the changes in 
+ * position and orientation of your mobile IO device.
+ */
+
+#include "group_command.hpp"
+#include "group_feedback.hpp"
+#include "robot_model.hpp"
+#include "arm/arm.hpp"
+#include "util/mobile_io.hpp"
+#include <chrono>
+#include <thread>
+#include "hebi_util.hpp"
+
+using namespace hebi;
+using namespace experimental; // For all things mobileIO 
+
+
+
+Eigen::Matrix3d makeRotationMatrix (hebi::Quaternionf phone_orientation) {
+  Eigen::Quaterniond q;
+  q.w() = phone_orientation.getW();
+  q.x() = phone_orientation.getX();
+  q.y() = phone_orientation.getY();
+  q.z() = phone_orientation.getZ();
+  return q.toRotationMatrix();
+}
+
+
+int main(int argc, char* argv[])
+{
+  //////////////////////////
+  ///// Config Setup ///////
+  //////////////////////////
+
+  // Config file path
+  const std::string example_config_file = "config/ex_AR_kit.cfg.yaml";
+  std::vector<std::string> errors;
+  
+  // Load the config
+  const auto example_config = RobotConfig::loadConfig(example_config_file, errors);
+  for (const auto& error : errors) {
+    std::cerr << error << std::endl;
+  }
+  if (!example_config) {
+    std::cerr << "Failed to load configuration from: " << example_config_file << std::endl;
+    return -1;
+  }
+
+  // For this demo, we need the arm and mobile_io
+  std::unique_ptr<hebi::experimental::arm::Arm> arm;
+  std::unique_ptr<hebi::util::MobileIO> mobile_io;
+
+  //////////////////////////
+  ///// Arm Setup //////////
+  //////////////////////////
+
+  // Create the arm object from the configuration
+  arm = hebi::experimental::arm::Arm::create(*example_config);
+
+  // Keep retrying if arm not found
+  while (!arm) {
+      std::cerr << "Failed to create arm, retrying..." << std::endl;
+
+      // Wait for 1 second before retrying
+      std::this_thread::sleep_for(std::chrono::seconds(1));  
+
+      // Retry
+      arm = hebi::experimental::arm::Arm::create(*example_config);
+  }
+  std::cout << "Arm connected." << std::endl;
+
+  //////////////////////////
+  //// MobileIO Setup //////
+  //////////////////////////
+
+  // Create the mobile_io object from the configuration
+  std::cout << "Waiting for Mobile IO device to come online..." << std::endl;
+  mobile_io = createMobileIOFromConfig(*example_config, example_config_file);
+
+  // Keep retrying if Mobile IO not found
+  while (mobile_io == nullptr) {
+      std::cout << "Couldn't find Mobile IO. Check name, family, or device status..." << std::endl;
+
+      // Wait for 1 second before retrying
+      std::this_thread::sleep_for(std::chrono::seconds(1));  
+
+      // Retry
+      mobile_io = createMobileIOFromConfig(*example_config, example_config_file);
+  }
+  std::cout << "Mobile IO connected." << std::endl;
+
+  // Setup instructions for display
+  std::string instructions;
+  instructions = ("🏠 - Home Position\n📲 - AR Control Mode\n"
+                  "🌍 - Grav Comp Mode\n❌ - Quit\n");
```

Comment:
Might as well define on the same line you declare here?

Reply:
Instructions have been removed entirely